common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalShuffleBlockResolver.java

```diff
@@ -368,7 +382,8 @@ public int removeBlocks(String appId, String execId, String[] blockIds) {
       if (file.delete()) {
         numRemovedBlocks++;
       } else {
-        logger.warn("Failed to delete block: " + file.getAbsolutePath());
+        logger.warn("Failed to delete block: {}",
```

Comment:
I'm not sure it's appropriate to call this `LogKeys.PATH$.MODULE$`?

Reply:
yes, path is fine